backend/package.json

```diff
@@ -11,33 +11,46 @@
     "migrate": "tsx src/db/migrate.ts",
     "test": "NODE_ENV=test mocha --require tsx/cjs 'src/**/*.test.ts' --file src/test/setup.ts",
     "test:watch": "mocha --watch",
+    "test:database": "mocha src/services/DatabaseService.test.ts",
+    "test:author": "mocha src/middleware/authMiddleware.test.ts",
+    "test:material.request": "mocha src/materials.test.ts",
     "test:storage": "mocha src/services/StorageService.test.ts"
   },
   "dependencies": {
     "@edge-prompt/common": "file:../common",
+    "@types/jsonwebtoken": "^9.0.9",
     "axios": "^1.6.7",
+    "bcrypt": "^5.1.1",
     "better-sqlite3": "^9.6.0",
     "cors": "^2.8.5",
     "dotenv": "^16.4.7",
     "express": "^4.18.2",
+    "jsonwebtoken": "^9.0.2",
     "mammoth": "^1.6.0",
     "multer": "^1.4.5-lts.1",
-    "pdfjs-dist": "3.11.174",
+    "pdfjs-dist": "^4.8.69",
```

Comment:
why changing the dependency version here, especially pdfjs-dist, which should be unrelated to auth?